client/ayon_hiero/api/pipeline.py

```diff
@@ -35,42 +51,80 @@
 AVALON_CONTAINERS = ":AVALON_CONTAINERS"
 
 
-def install():
-    """Installing Hiero integration."""
 
-    # adding all events
-    events.register_events()
+class HieroHost(
+    HostBase, IWorkfileHost, ILoadHost, IPublishHost
+):
+    name = "hiero"
 
-    log.info("Registering Hiero plug-ins..")
-    pyblish.register_host("hiero")
-    pyblish.register_plugin_path(PUBLISH_PATH)
-    register_loader_plugin_path(LOAD_PATH)
-    register_creator_plugin_path(CREATE_PATH)
+    def open_workfile(self, filepath):
+        return open_file(filepath)
 
-    # register callback for switching publishable
-    pyblish.register_callback("instanceToggled", on_pyblish_instance_toggled)
+    def save_workfile(self, filepath=None):
+        return save_file(filepath)
 
-    # install menu
-    menu.menu_install()
-    menu.add_scripts_menu()
+    def work_root(self, session):
+        return work_root(session)
 
-    # register hiero events
-    events.register_hiero_events()
+    def get_current_workfile(self):
+        return current_file()
 
+    def workfile_has_unsaved_changes(self):
+        return has_unsaved_changes()
 
-def uninstall():
-    """
-    Uninstalling Hiero integration for avalon
+    def get_workfile_extensions(self):
+        return file_extensions()
 
-    """
-    log.info("Deregistering Hiero plug-ins..")
-    pyblish.deregister_host("hiero")
-    pyblish.deregister_plugin_path(PUBLISH_PATH)
-    deregister_loader_plugin_path(LOAD_PATH)
-    deregister_creator_plugin_path(CREATE_PATH)
+    def get_containers(self):
+        return ls()
+
+    def install(self):
+        ''' Installing all requarements for hiero host
+        '''
+
+        # adding all events
+        events.register_events()
+
+        log.info("Registering Hiero plug-ins..")
+        pyblish.register_host("hiero")
+        pyblish.register_plugin_path(PUBLISH_PATH)
+        register_loader_plugin_path(LOAD_PATH)
+        register_creator_plugin_path(CREATE_PATH)
+
+        # register callback for switching publishable
+        pyblish.register_callback(
+            "instanceToggled", on_pyblish_instance_toggled)
+
+        # install menu
+        menu.menu_install()
+        menu.add_scripts_menu()
+
+        # register hiero events
+        events.register_hiero_events()
+
+    def uninstall(self):
+        """
+        Uninstalling Hiero integration for avalon
+
+        """
```

Comment:
```suggestion
        """Uninstalling Hiero integration for AYON"""
```

At first I thought `uninstall()` was never used anywhere, but apparently it still lives [here](https://github.com/ynput/ayon-core/blob/717e370be56714a2c917ad9f7c3cae38b860995a/client/ayon_core/pipeline/context_tools.py#L245). I guess we _should_ actually implement that on `HostBase` too just so the host class defines its interface better @iLLiCiTiT ?

Reply:
Is there any particular reason we still need it in AYON? @iLLiCiTiT